tests/acceptance/conftest.py

```diff
@@ -28,24 +28,25 @@ def pytest_configure(config):
             last_built = int(time.time()) - data["built"]
 
             if last_built <= 3600:
-                print (  # noqa: B314
+                print (
                     """
 ###################
 #
-# Frontend assets last built %d seconds ago, skipping rebuilds for another %d seconds.
+# Frontend assets last built {} seconds ago, skipping rebuilds for another {} seconds.
```

Comment:
If this changed to `str.format()` we need to change it to an unicode literal. Or just leave it how it was.

Granted, the change here would be fine, but I tried to make a pass of removing all `str.format()` calls everywhere since it causes problems in python2.

Reply:
> str.format() calls everywhere since it causes problems in python2.

Can you please provide an example?

Reply:
Matt is talking about how unicode is implicitly coerced with str.format in Python 2.

'foo'.format(u'insert non ascii')

vs

u'foo'.format('insert non ascii')

Reply:
```python
"{}".format(u"ã")
```

We’ve historically had dozens of errors related to str.format vs unicode.format. I understand this isn’t a problem in python3 since they gratefully and thankfully removed the ability to format bytestrings. And keep it only to unicodes. But until then, would prefer we be explicit on unicode strings when we use .format() to avoid issues.

Reply:
e597c9c